static/elements/form-field-specs.js

```diff
@@ -1127,9 +1122,7 @@ function makeDisplaySpecs(fields) {
 };
 
 export const DISPLAY_FIELDS_IN_STAGES = {
-  'Metadata': makeDisplaySpecs([
-    'category', 'feature_type', 'intent_stage', 'accurate_as_of',
-  ]),
+  'Metadata': makeDisplaySpecs(['category', 'intent_stage', 'accurate_as_of']),
```

Comment:
I think we should still display the feature_type.

Reply:
Agreed. I've added  a change to display it as a disabled field. 👍